cds_ils/importer/views.py

```diff
@@ -0,0 +1,171 @@
+        return unique_filename + "." + ext
+
+    @blueprint.errorhandler(413)
+    def payload_too_large(error):
+        return {"message": "The file is too large", "status": 413}, 413
+
+    @blueprint.route("/importer/check/<int:log_id>", methods=["GET"])
+    @need_permissions("document-importer")
+    def check(log_id):
+        return check_next(log_id, 0)
+
+    @blueprint.route("/importer/check/<int:log_id>/next/<int:next_entry>",
+                     methods=["GET"])
+    @need_permissions("document-importer")
+    def check_next(log_id, next_entry):
+        log = ImporterTaskLog.query.filter_by(id=log_id).first()
+        if log:
+            children_entries_query = ImporterTaskEntry.query \
+                .filter_by(import_id=log_id)
+            entries = children_entries_query \
+                .filter(ImporterTaskEntry.entry_index >= next_entry) \
+                .order_by(ImporterTaskEntry.entry_index.asc()) \
+                .all()
+            obj = {
+                "id": log_id,
+                "state": log.status.value,  # enum value
+                "start_time": arrow.get(log.start_time).isoformat(),
+                "end_time": arrow.get(log.end_time).isoformat() \
+                if log.end_time else None,
+                "original_filename": log.original_filename,
+                "provider": log.provider,  # string
+                "mode": log.mode.value,  # enum value
+                "source_type": log.source_type,  # string
+            }
+            if log.entries_count:
+                obj["total_entries"] = log.entries_count
+                obj["loaded_entries"] = children_entries_query.count()
+            reports = []
+            for entry in entries:
+                if not entry.error:
+                    reports.append({
+                        "index": entry.entry_index,
+                        "success": True,
+                        "report": {
+                            "ambiguous_documents": entry.ambiguous_documents,
+                            "ambiguous_eitems": entry.ambiguous_eitems,
+                            "created_document": entry.created_document,
+                            "created_eitem": entry.created_eitem,
+                            "updated_document": entry.updated_document,
+                            "updated_eitem": entry.updated_eitem,
+                            "deleted_eitems": entry.deleted_eitems,
+                            "series": entry.series,
+                            "fuzzy_documents": entry.fuzzy_documents,
+                        }
+                    })
+                else:
+                    reports.append({
+                        "index": entry.entry_index,
+                        "success": False,
+                        "message": entry.error,
+                    })
+            obj["reports"] = reports
+            return obj
+        else:
+            return 404
+
+    @blueprint.route("/importer", methods=["POST"])
```

Comment:
```suggestion
    @blueprint.route("/importer/tasks", methods=["POST"])
```
to be added to the ticket?